handbook/people-ops/hiring.md

```diff
@@ -56,3 +56,37 @@ If any interviewer is asked by a candidate for interview feedback, they may resp
 ## Respecting time
 
 Interviewing requires time and effort, both for us and for candidates. If we determine that we won't be moving forward in our hiring process before a candidate has completed all interviews, then we will cancel the remaining interviews to be respectful of both the candidate's and interviewers' time.
+
+## Referrals
+
+If there is someone out there that you would be excited to work with everyday– we encourage you to refer them! 
+
+Perhaps they are a friend or maybe they’re a friend of a friend. Perhaps they’re a former classmate or someone you worked with in the past. Maybe you posted our open roles in your social network and they decided to apply when they saw it. 
+
+We welcome referrals in all forms. However, referrals fall into two categories, depending on how actively you were involved in their decision to apply to Sourcegraph:
+
+* **First-hand referral:** A first-hand referral is someone you have personally had significant interactions with. You have met them and chatted with them on more than one occasion, specifically about opportunities at Sourcegraph. Referral bonus is USD $5,000.
```

Comment:
I believe referral bonuses will be counterproductive for our team and unique culture. I think it's great to recognize people who help grow our team, but this makes it feel like it's not intrisically motivated/conflict of interest. I would be in favor of recognizing people with general bonuses who have helped us grow our team, but not having such direct linkage.